build/build.sh

```diff
@@ -63,9 +63,22 @@ yarn install
   yarn install
   npm rebuild node-sass
   node extractLocals.js
-  node_modules/.bin/node-sass --output dist/css --sourcemap=none scss/
+  node_modules/.bin/node-sass --sourcemap=none scss/all.scss dist/themes/light.css
```

Comment:
I'm proposing to not make it an external module but instead put the code into the project.